metagraph-katana-plugin/git_push.sh

```diff
@@ -0,0 +1,3 @@
+git add .
```

Comment:
Let's not include these scripts. We don't use scripts like this.

Reply:
I'm guessing you use them locally which is fine. Just don't commit them to the repo.